app/src/main/java/com/safetyheads/akademiaandroida/MainActivity.kt

```diff
@@ -87,36 +83,36 @@ class MainActivity : AppCompatActivity() {
         )
 
         //create instance Remote Config to firebase and download data
-        val remoteConfig = FirebaseRemoteConfig.getInstance()
-        val configSettings = FirebaseRemoteConfigSettings.Builder()
-            .setFetchTimeoutInSeconds(60)
-            .build()
-        remoteConfig.setConfigSettingsAsync(configSettings)
-        remoteConfig.setDefaultsAsync(R.xml.remote_config_defaults)
-
-        val versionCode = remoteConfig.getString("versionCode")
-        val apiUrl = remoteConfig.getString("apiUrl")
-
-        remoteConfig.fetchAndActivate()
-            .addOnCompleteListener(this) { task ->
-                if (task.isSuccessful) {
-                    val updated = task.result
-                    Log.d(TAG, "Config params updated: $updated")
-
-                    Toast.makeText(
-                        this,
-                        "versionCode:" + versionCode +
-                                "apiUrl:" + apiUrl,
-                        Toast.LENGTH_LONG
-                    ).show()
-
-                } else {
-                    Toast.makeText(
-                        this, "Fetch failed",
-                        Toast.LENGTH_SHORT
-                    ).show()
-                }
-            }
+//        val remoteConfig = FirebaseRemoteConfig.getInstance()
```

Comment:
Are you sure this should be commented out and not removed?

Reply:
Oh no, I completely forgot about it
Of course it should be removed